Install jenv to handle JDK versions (#94)
faberNovel/docker-android: README.md

- Comment https://github.com/faberNovel/docker-android/issues/78 so that we can
keep track of missing native libraries and add them to the next image version.

## JDK support

> Images support multiple JDK, using [Jenv](https://www.jenv.be/).
> By default JDK 11 is used, but JDK 8 is also supported (use `jenv global 1.8`).
> To choose a JDK, make sure the script executes on a bash shell, then use Jenv with the desired version: 

Reviewer: ```suggestion
Images support multiple JDK using [Jenv](https://www.jenv.be/).
The default JDK is JDK 11 is used, but JDK 8 is also supported (use `jenv global 1.8`).
To choose a JDK, make sure the script executes on a bash shell, then use Jenv to set the desired version: 
```
